packages/form-js-viewer/test/spec/render/components/FormField.spec.js

```diff
@@ -518,6 +519,12 @@ function createFormField(options = {}) {
             return isExpression(...args);
           }
         } : undefined;
+      } else if (type === 'templating') {
+        return isTemplate !== false ? {
+          isTemplate(...args) {
+            return isTemplate(...args);
+          }
+        } : undefined;
```

Comment:
```suggestion
        return isTemplate ? {
          isTemplate(...args) {
            return isTemplate(...args);
          }
        } : undefined;
```

Reply:
We use this pattern to test `undefined` services, we already had a similar one for the `conditionChecker` service. It could always happen that you misconfigure the viewer and certain services are not available, so we want to make sure we do not blow up.

I added a test case for `templating` as well.